nvi

```diff
@@ -10,7 +10,12 @@ N=$(tput sgr0) # Normal
 U=$(tput smul) # Underline
 
 print_usage() {
-  cat << EOF
+  less \
+    --raw-control-chars \
+    --no-init \
+    --quit-if-one-screen \
+    --QUIT-AT-EOF \
```

Comment:
It's a very good point about the distinction between `nvi` on host systems and CI environments, and I think that's the key to a good solution. It just struck me that we are not limited to providing just one format of `nvi` as we'll be delivering them through GitHub releases where you can have any number of files and formats for a single release.

What do you think about the following suggestion?

Releases of `nvi` come in two formats:

1.  A single file, `nvi`, which can be directly downloaded without archive wrapping, and which gives a rudimentary `--help`, but doesn't work with `man`.
2.  A fully-fledged archive containing `nvi` and whatever files we need to make `man` work properly.

This allows CI environments to download just the executable, while local development use can benefit from the full man page.

I think this is the most appealing solution.